symfit/core/fit.py

```diff
@@ -1390,15 +1390,15 @@ def _determine_objective(model, objective, minimizer, bound_arguments):
                 # be considered separately and has its own non standard
                 # objective function.
                 return VectorLeastSquares
-            if len(model) == 1 and len(model.independent_vars) == 0:
-                if model.dependent_vars[0].name not in bound_arguments.arguments:
-                    # No data provided means a simple minimization of the Model
-                    # parameters is requested, not a fit. Therefore set the
-                    # variable to None and return MinimizeModel.
-                    for var in model.dependent_vars:
-                        if var.name not in bound_arguments.arguments:
-                            bound_arguments.arguments[var.name] = None
-                    return MinimizeModel
+            if (len(model) == 1 and len(model.independent_vars) == 0 and
+                model.dependent_vars[0].name not in bound_arguments.arguments):
```

Comment:
Except that now the continuation line is underindented and looks like the body of the if.